### PR TITLE
Updated snow crash and released a new version

### PR DESCRIPTION
**Pull request by pksunkara**
@zdne @klokane @kylef Review & Merge please.

### CODE REVIEW

File: test/sourcemap-test.coffee [hunk 1]
```diff
@@ -27,6 +27,6 @@ describe "Parser sourcemap", ->
 
   # Parser Sourcemap should conform to recent source map serialization JSON media type
   it '`sourcemap` field conforms to `vnd.apiblueprint.sourcemap+json; version=2.1`', ->
-    console.log JSON.stringify sourcemap_parsed, undefined, 2
+    # console.log JSON.stringify sourcemap_parsed, undefined, 2
```
Comment (jsarenik) [inline test/sourcemap-test.coffee]: Why this?

Comment (pksunkara) [inline test/sourcemap-test.coffee]: This was not commented in the original code by accident. When running tests, this prints a lot of lines thus distracting me from the actual test report. So, I commented it out.

**Comment by jsarenik** [comment]
Good to go :+1:

**Comment by jsarenik** [comment]
Any Trello card next time?

**Comment by pksunkara** [comment]
No trello card for this one.

**Comment by zdne** [comment]
@pksunkara correct answer for @jsarenik is that we have decided to not attach cards to open source PRs even if there is one

**Comment by zdne** [comment]
@pksunkara also please note, and I have said it already elsewhere, do not mention ppl in PR unless it is specifically for them (besides you want to mention whole team on every PR ? :)

**Comment by jsarenik** [comment]
OK. Thanks for explanation.

**Pull request merged**